Travis/callTomcat.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```

Comment:
Yes. This file is needed in case we want to run all services with gui tests. 
By this we check that all services are running. You can see that in pom.xml.